pkg/model/types.go

```diff
@@ -62,6 +63,7 @@ type Port struct {
 // Cloudmap Instances IP and Port is supposed to be AWS_INSTANCE_IPV4 and AWS_INSTANCE_PORT
 // Rest are custom attributes
 const (
+	CreationTimestampAttr = "CREATION_TIMESTAMP"
```

Comment:
This is a confusing name the attribute, since it means export time not endpoint create time.

Reply:
True. What about `SvcExportCreationTimestampAttr = "SVC_EXPORT_CREATION_TIMESTAMP"`?

Reply:
Works for me.